src/ONGR/DemoBundle/Resources/doc/index.rst

```diff
@@ -55,15 +55,7 @@ Let's rock. Move into your project root folder and execute:
 
 ..
 
-(In case you have also something like VMWare installed on your local machine, it is a good idea to give the provider when upping your box:
-
-.. code-block:: bash
-
-    vagrant up --provider=virtualbox
-
-..
-
-)
+.. note:: Currently we have only Virtualbox provider support. We are working on others and soon should be docker provider ready to use.
```

Comment:
Are you sure that RTD will render it ok, if its on the same line? Docs says that it should be on the next line: 
http://sphinx-doc.org/markup/para.html